Extensions/DataSources/src/DataSourcesHub/Resources/DataSourceResources.tsx

```diff
@@ -0,0 +1,7 @@
+export const SUBMIT = 'SUBMIT';
```

Comment:
wrap all these in Resources class
also don't name them like DATA_SOURCES, name them like DataSources